cli/helm.go

```diff
@@ -15,32 +15,32 @@ import (
 // and should not be trusted.
 var version = "0.1.0"
 
-const globalUsage = `The Kubernetes package manager
+const globalUsage = `Helm Classic - The original Kubernetes package manager
```

Comment:
I was only trying to match it to the documentation.  We've used that tagline in a few places to try to distinguish Helm Classic from the new Helm, since this is something that as confused the community a bit up until now.  If we change this here, we should probably change it in other places as well.  Let's run this by @slack before I merge.